k8s/docs/values-stage.yaml

```diff
@@ -1,13 +1,9 @@
-# Default values for airqo-docs.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-
 replicaCount: 1
 
 image:
   repository: gcr.io/airqo-250220/airqo-stage-docs
   pullPolicy: Always
-  tag: latest
+  tag: stage-090ad16f-1667922055
```

Comment:
What is the motivation for these tags?

Reply:
@Baalmart, we have an auto update bot that updates those images.
This is important to trigger ArgoCD as it only watches for changes in the helm files.

Reply:
Alright @123MwanjeMike , thanks for the clarifications.

Is this related to image versioning? If yes, what is unique about this tag number and how is differentiated from "previous" tags?